src/Tools/Microsoft.CodeAnalysis.Toolset.Open/Targets/VSL.Settings.targets

```diff
@@ -58,17 +58,27 @@
     <RoslynDiagnosticsPropsFilePath>$(NuGetPackagesPath)\Microsoft.Net.RoslynDiagnostics.1.0.0-rc2-20150313-01\build\Microsoft.Net.RoslynDiagnostics.props</RoslynDiagnosticsPropsFilePath>
     <AdditionalFileItemNames>$(AdditionalFileItemNames);PublicAPI</AdditionalFileItemNames>
   </PropertyGroup>
-  <ImportGroup Label="GlobalNuGets">
+
+  <!-- Import the props files from the toolset NuGet packages if we're not in a bootstrap build -->
+  <ImportGroup Label="GlobalNuGets" Condition="'$(BootstrapBuild)' == ''">
```

Comment:
'$(BootstrapBuild)' != 'false' would be the idiomatic form. Otherwise this would do a bootstrap build if you said to not do one.